2-api/models.md

```diff
@@ -2,35 +2,39 @@
 
 # Models and attributes
 
-This section details Perspective API’s production and experimental models, which are described in detail in the [All model types](#all-model-types) table below.
+A model is a set of attributes (sometimes called "model attributes") on which a comment is scored on. For example, the Toxicity model contains attributes such as "Toxcity", "Profanity", "Fliratation", and more. [See all model types](#all-model-types). Models can be in-production or experimental.
```

Comment:
Discussed with eng offline - let's remove mention of "models" and just call them attributes, especially since the field is called requestedAttributes in the API request. Then we can avoid having to explain the relationship between these terms. We'll want to update this in a few different sections here, and in the Methods doc as well.